int64 floor
int64 box_num
---
int64 res